src/index.js

```diff
@@ -1,9 +1,20 @@
 // ITERATION 1
 
 function updateSubtotal(product) {
-  console.log('Calculating subtotal, yey!');
-
-  //... your code goes here
+  console.log('Calculating subtota, yey!');
+  const quantity = product.querySelector('.quantity span');
```

Comment:
Inside quantity we don't have a `span` we actually have an `input`, and get that `value` would be a little bit different